src/main/resources/templates/java/test/projectTemplate/pom.xml

```diff
@@ -61,6 +61,17 @@
                 <artifactId>maven-failsafe-plugin</artifactId>
                 <version>2.22.2</version>
             </plugin>
+            %static-code-analysis-start%
```

Comment:
Is it planned to only enable some of the static code analysis tools (e.g. only SpotBugs and not a different tool)?
This cannot be realized well here, so it might be better to use something like `static-code-analysis-stopbugs-start` instead?

Reply:
I have no concept for this at the moment. We probably just enable all tools for a language and provide some predefined rule configurations for instructor. I think this is fine and we can adapt this if we have a concept for configurations.